### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by ma-oliveiramarques**
Check out if I'm good please :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,11 +8,18 @@ This respository aims to create a calculator, made in vanilla JavaScript with un
...
+- [Node.js](https://nodejs.org/en/docs/)
 - [Jest](https://jestjs.io)
 
+## External packages
+
+### `import rl from 'readline';`
+
+Readline Module in Node.js allows the reading of input stream line by line.
+
+[Readline package](https://www.geeksforgeeks.org/node-js-readline-module/)
```
Comment (pncsoares) [inline README.md:21]: I suggest to leave the name as Documentation only instead of the custom name Readline package

File: src/all-operations.js [hunk 2]
```diff
@@ -1,10 +1,24 @@
+/**
+ * import readline package from node.js
```
Comment (pncsoares) [inline src/all-operations.js:2]: This is unnecessary

**Review by pncsoares — COMMENTED** [review]
Please consider what we talked about in today's meeting

**Comment by pncsoares** [comment]
> Check out if I'm good please :)

It's all the same as far as I could understand

File: README.md [hunk 3]
```diff
@@ -8,11 +8,18 @@ This respository aims to create a calculator, made in vanilla JavaScript with un
 
 # Technologies
 
-- [HTML](https://www.w3schools.com/html/)
-- [CSS](https://www.w3schools.com/css/)
 - [JavaScript](https://www.javascript.com)
```
Comment (pncsoares) [inline README.md:13]: This is not the documentation page

File: README.md [hunk 4]
```diff
@@ -8,11 +8,18 @@ This respository aims to create a calculator, made in vanilla JavaScript with un
...
+- [Node.js](https://nodejs.org/en/docs/)
 - [Jest](https://jestjs.io)
```
Comment (pncsoares) [inline README.md:14]: This is not the documentation page

Comment (pncsoares) [inline README.md]: This is not the documentation page

File: README.md (hunk 3 above)
Comment (pncsoares) [inline README.md]: This is not the url

File: src/all-operations.js [hunk 5]
```diff
@@ -1,10 +1,24 @@
...
+/**
+ * @typedef {number} firstNumber
+ * @typedef {number} secondNumber
+ * @typedef {number} result
+ */
+
+/**
+ * Function responsable for displaying the questions to the user and receive the user's input
+ * 
+ */
```
Comment (pncsoares) [inline src/all-operations.js:21]: You don't need to document a library that is already documented by the creators

**Review by pncsoares — CHANGES_REQUESTED** [review]

**Review by pncsoares — APPROVED** [review]

**Comment by ma-oliveiramarques** [comment]
Close #14

**Pull request merged**